VKClient/VKClient/Business Logic/Network/GroupSession.swift

```diff
@@ -0,0 +1,43 @@
+// GroupSession.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+import RealmSwift
+import SwiftyJSON
+
+final class GroupSession: AsyncOperation {
+    // MARK: - Private Properties
+
+    private let token = Session.shared.token
+    private let baseURL = "https://api.vk.com"
+    private let version = "5.131"
+
+    // MARK: - Public Methods
+
+    public func getGroups() {
+        let path = "/method/groups.get"
+        guard let token = token else { return }
+        let parameters: Parameters = [
+            "access_token": token,
+            "v": version,
+            "extended": "1"
+        ]
+
+        let url = baseURL + path
+
+        let operationQueue = OperationQueue()
```

Comment:
```suggestion
        let operationQueue = OperationQueue()
```
Я думаю ее можно к остальным константам вынести...